reactApp2/child.jsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 
 class Child extends React.Component { 
   constructor(props) {
     super(props);
-    this.state = {array: [], filterArray: [], state: ""};
+    this.state = {array: [], state: ""};
```

Comment:
@VARTIKASINGH1234 modify the 'state' name to 'selectedFilter' & array to 'nameList'